fastlane_bot/helpers/txhelpers.py

```diff
@@ -223,9 +223,14 @@ def _get_prices_info(
             / Decimal("10") ** Decimal("18")
         )
 
-        if self.ConfigObj.network.GAS_ORACLE_ADDRESS:
-            layer_one_gas_fee = self._get_layer_one_gas_fee_loop(signed_arb_tx)
-            gas_cost_eth += layer_one_gas_fee
+        if self.ConfigObj.NETWORK in ["optimism", "coinbase_base"]:
```

Comment:
I don't know, but `self.ConfigObj.network.GAS_ORACLE_ADDRESS` is defined for both of them, and since it now needs to be defined also for Mantle, the only way to separate between them (Optimism and Base vs Mantle) is by network, and not by the viability of that contract.